app/src/main/java/ai/brokk/gui/git/GitCommitBrowserPanel.java

```diff
@@ -524,7 +524,7 @@ private void setupCommitContextMenu() {
         registerMenu(commitsContextMenu);
 
         addToContextItem = new JMenuItem("Capture Diff");
-        captureWorkspaceSelectionsItem = new JMenuItem("Capture workspace selections at this revision");
+        captureWorkspaceSelectionsItem = new JMenuItem("Capture workspace content at this revision");
```

Comment:
I think this menu item text is descriptive enough, now we capture the revision of all project files that are in the context, as there is no (multi-)select support on chips panel.